lambda_function.py

```diff
@@ -1,9 +1,161 @@
 import json
+import boto3
+import time
+import logging
 
-print('Loading function')
+from dome9_type_annotations.client import Client
+from resources.aws_cloud_account import CloudAccount, CloudAccountCredentials
+from uuid import uuid4
+from typing import Dict
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+class Onboarder(object):
+
+    MASTER_ACCOUNT_PERMISSIONS_STACK_SET_NAME = 'Dome9AutomaticOnboardigStackSet'
+    CUSTOMER_ACCOUNT_EXECUTION_ROLE_NAME = "AWSControlTowerExecution"
+    MASTER_ACCOUNT_STACK_SET_ROLE = "service-role/AWSControlTowerStackSetRole"
+    STACK_OPERATION_WAIT_RETRIES = 12
+    STACK_OPERATION_WAIT_SLEEP = 10
+
+    def __init__(self, region_name: str, customer_account_id: str, customer_account_name: str, readonly: bool = True) -> None:
+        logger.info(f"Initing onboarder with region_name: '{region_name}', customer_account_id: '{customer_account_id}', customer_account_name: '{customer_account_name}'")
```

Comment:
we use single quote. in such case we do:
`logger.info(f'''my name is: '{name}' and i rock''')`